test/metabase/http_client.clj

```diff
@@ -44,7 +45,7 @@
   #{:created_at :updated_at :last_login :date_joined :started_at :finished_at :last_analyzed})
 
 (defn- auto-deserialize-dates
-  "Automatically recurse over `response` and look for keys that are known to correspond to dates. Parse their values and
+  "Automatically recurse over `response` and look for values that are known to correspond to dates. Parse their values and
```

Comment:
I think the word `keys` is correct in this sentence, we actually look for `keys` and parse their `values`

Reply:
oh, you're right, good catch. reverting